packages/fxa-admin-panel/src/components/PageAccountSearch/ConnectedServices/index.tsx

```diff
@@ -0,0 +1,162 @@
+            {...service}
+          />
+        ))}
+      </>
+    );
+
+    if (services.length > NUMBER_OF_SERVICES_TO_SHOW) {
+      return (
+        <details>
+          <summary className="hover:cursor-pointer text-violet-900 font-semibold mb-4">
+            Toggle viewing {services.length} connected services
+          </summary>
+          {connectedServicesTables}
+        </details>
+      );
+    }
+    return connectedServicesTables;
+  }
+
+  return <p className="result-none">This account has no connected services.</p>;
+};
+
+const ConnectedService = ({
+  clientId,
+  createdTime,
+  createdTimeFormatted,
+  deviceId,
+  deviceType,
+  lastAccessTime,
+  lastAccessTimeFormatted,
+  location,
+  name,
+  os,
+  userAgent,
+  sessionTokenId,
+  refreshTokenId,
+}: AttachedClient) => {
+  const testId = (id: string) => `connected-service-${id}`;
+  return (
+    <TableYHeaders>
+      <TableRowYHeader
+        header="Client"
+        value={format.client(name, clientId)}
+        testId={testId('client')}
+      />
+      <TableRowYHeader
+        header="Device Type"
+        value={deviceType}
+        testId={testId('device-type')}
+      />
+      <TableRowYHeader
+        header="User Agent"
+        value={userAgent}
+        testId={testId('user-agent')}
+      />
+      <TableRowYHeader
+        header="Operating System"
+        value={os}
+        testId={testId('os')}
+      />
+      <TableRowYHeader
+        header="Created At"
+        value={format.time(createdTime, createdTimeFormatted)}
+        testId={testId('created-at')}
+      />
+      <TableRowYHeader
+        header="Last Used"
+        value={format.time(lastAccessTime, lastAccessTimeFormatted)}
+        testId={testId('last-accessed-at')}
+      />
+      <TableRowYHeader
+        header="Location"
+        value={format.location(location)}
+        testId={testId('location')}
+      />
+      <TableRowYHeader
+        header="Client ID"
+        value={clientId}
+        testId={testId('client-id')}
+      />
+      <TableRowYHeader
+        header="Device ID"
+        value={deviceId}
+        testId={testId('device-id')}
+      />
+      <TableRowYHeader
+        header="Session Token ID"
+        value={sessionTokenId}
+        testId={testId('session-token-id')}
+      />
+      <TableRowYHeader
+        header="Refresh Token ID"
+        value={refreshTokenId}
+        testId={testId('refresh-token-id')}
+      />
+    </TableYHeaders>
+  );
+};
+
+const format = {
+  location(location?: Nullable<Location>) {
+    if (
+      !location ||
+      (!location.city &&
+        !location.state &&
+        !location.stateCode &&
+        !location.country &&
+        !location.countryCode)
+    ) {
+      return null;
+    }
+
+    return (
+      <>
+        {[
+          location.city || <i>Unknown City</i>,
+          location.state || location.stateCode || '',
+          location.country || location.country || <i>Unknown Country</i>,
```

Comment:
`location.country || location.country`

Reply:
I am guessing it was supposed to be countryCode